pkg/virt-launcher/virtwrap/network/common_test.go

```diff
@@ -149,3 +152,49 @@ func createDummyVIF(vifName, ipv4cidr, ipv4gateway, ipv6cidr, macStr string, mtu
 
 	return vif
 }
+
+var _ = Describe("infocache", func() {
```

Comment:
I'm not so sure that it is a good idea to mandate a 1:1 mapping between production files and test files. Where I place the production code is an implementation detail that should not affect where I place the test.

I need to keep the new tests in the `network` package (which is a sin on its own right) because I am testing private function, and using an ugly private mock function. These are things that I hope to fix in follow-up PRs. It would be very easy to add another file in the `network` package, but I am reluctant to do so as it "spreads the sin" further.